src/styles/card.css

```
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  align-self: center;
+}
+
+.card__lable {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main_color);
+  padding-top: 40px;
+}
+
+.card__code,
+.review__text,
+.card__price-lable {
+  font-weight: 400;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary_color);
+  padding-top: 4px;
+}
+
+.review__stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 16px;
+  gap: 17px;
+}
+
+.review__text {
+  font-size: 10px;
+  line-height: 14px;
+  align-self: flex-end;
+  color: var(--main_color);
+}
+
+.card__price-area {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 24px;
+  padding-bottom: 16px;
+}
+
+.card__price-lable {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary_color);
+}
+
+.card__price,
+.card__buy-product {
+  font-weight: 700;
+}
+
+.card__price {
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main_color);
+}
+
+.card__buy-product {
+  display: flex;
+  background: var(--price_color);
+  border: 1px solid transparent;
+  border-radius: 5px;
+  color: white;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+  padding: 11px 69px 11px 68px;
```

it should have been something like this, you're missing a pixel somewhere
```suggestion
  padding: 11px 69px;
```